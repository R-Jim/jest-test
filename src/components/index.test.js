import renderer from 'react-test-renderer';
import DivCustom from '.';

// NOTES: Reset snapshot with yarn jest --updateSnapshot 

// Test snapshot
it('renders correctly', () => {
  const tree = renderer
    .create(DivCustom)
    .toJSON();
  expect(tree).toMatchSnapshot();
});