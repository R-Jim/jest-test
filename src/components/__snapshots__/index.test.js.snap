// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  url="http://www.facebook.com"
>
  Facebook
</div>
`;
